ConsoleHW7/AssemblyOne/Developer.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AssemblyOne
+{
+    class Developer : Employee
+    {
+        private const string position = "Developer";
```

Comment:
All const variable should be started with Upper case letter, 
for example
private const string position => `private const string Position`

Reply:
all consts a renamed with Upper case latter